ui/decredmaterial/image.go

```diff
@@ -30,14 +30,15 @@ func (t *Theme) Image(img paint.ImageOp) Image {
 }
 
 func (im Image) Layout(gtx layout.Context) layout.Dimensions {
-	size := im.Src.Rect.Size()
+	size := im.Src.Size()
 	wf, hf := float32(size.X), float32(size.Y)
 	w, h := gtx.Px(unit.Dp(wf*im.Scale)), gtx.Px(unit.Dp(hf*im.Scale))
 	d := gtx.Constraints.Constrain(image.Point{X: w, Y: h})
-	var s = op.Push(gtx.Ops)
+	var s = op.Save(gtx.Ops)
 	clip.RRect{Rect: f32.Rectangle{Max: toPointF(d)}}.Add(gtx.Ops)
 	im.Src.Add(gtx.Ops)
-	paint.PaintOp{Rect: f32.Rectangle{Max: f32.Point{X: float32(w), Y: float32(h)}}}.Add(gtx.Ops)
-	s.Pop()
+	//paint.PaintOp{Rect: f32.Rectangle{Max: f32.Point{X: float32(w), Y: float32(h)}}}.Add(gtx.Ops)
+	paint.PaintOp{}.Add(gtx.Ops)
+	s.Load()
```

Comment:
this looks like a direct copy of widget.Image with no added feature. It can be removed and widget.Image used instead.